### ISSUE TITLE
different disassemble results compared to objdump for "\x66\x90"

### ISSUE THREAD

**Issue by zhouxucs** [aquynh/capstone#1177]
In my test, Capstone decodes "\x66\x90" as "nop" while objdump decodes it as "xchg   %ax,%ax" . I am not sure which one is correct, though both instructions have not effect on CPU state.

**Comment by aquynh** [comment]
Both are correct, because they are all doing nothing.

**Comment by zhouxucs** [comment]
Of course. However, as far as I know, some emulators such as Simics adopts "xchg %ax, %ax" as magic instructions, because compiler does not generate these instructions. However, compiler does generate "nop" instructions.

**Comment by ooyyloo** [comment]
I wonder, is there any way to ouput "xchg %ax, %ax" instead of "nop" using capstone when decoding "\x66\x90".
